webassembly/api.json

```
@@ -90,6 +90,50 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "compile_options": {
+          "__compat": {
+            "description": "`compileOptions` parameter",
+            "spec_url": "https://github.com/WebAssembly/js-string-builtins/blob/main/proposals/js-string-builtins/Overview.md#using-builtins",
+            "tags": [
+              "web-features:wasm"
```

I created a web-feature for this recently. https://github.com/web-platform-dx/web-features/blob/main/features/wasm-string-builtins.yml

```suggestion
              "web-features:wasm-string-builtins"
```